prowler/providers/aws/services/cognito/cognito_user_pool_strong_password_policy/cognito_user_pool_strong_password_policy.py

```diff
@@ -0,0 +1,39 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.cognito.cognito_idp_client import cognito_idp_client
+
+
+class cognito_user_pool_strong_password_policy(Check):
+    def execute(self):
+        findings = []
+        for pool in cognito_idp_client.user_pools.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = pool.region
+            report.resource_name = pool.name
+            report.resource_id = pool.id
+            report.resource_arn = pool.arn
+            report.resource_tags = pool.tags
+            if pool.password_policy:
+                if (
+                    pool.password_policy.get("RequireLowercase", False)
+                    and pool.password_policy.get("RequireUppercase", False)
+                    and pool.password_policy.get("RequireNumbers", False)
+                    and pool.password_policy.get("RequireSymbols", False)
+                    and pool.password_policy.get("MinimumLength", 8) >= 14
```

Comment:
Please, can you create a check for each property like in IAM? Thanks!

Reply:
Done